README.md

```diff
@@ -40,23 +41,23 @@ GMAIL_PASSWORD=password
 ```
 
 For Ruby scripts you need to install gems:
-`gem install dotenv twilio-ruby gmail`
+`gem install dotenv Twilio-ruby Gmail`
```

Comment:
This change is unnecessary.

Reply:
depends, are gem packages case sensitive?

Reply:
cool